src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java

```diff
@@ -114,6 +114,11 @@
     @Parameter(name = "validate", defaultValue = "true", property = "validate")
     protected boolean validate;
 
+    /**
+     * The value that will be stripped off of the groupId and used in path setting and job names.
+     */
+    @Parameter(name = "prefixToStrip", property = "prefixToStrip", defaultValue = "com\\.edmunds\\.")
```

Comment:
I suppose the defaultValue should just be com\\. OR perhaps even nothing at all...
(then in our dwh-defaults bom we can set it to our edmunds value)

Reply:
its your call at the end of the day! If we don't do it as part of this PR, I would say a separate issue to modify it? I am fine with that. At the end of the day, I think we agree that we don't want edmunds to be a default value anywhere.

Reply:
Ok, I'm leaving this.